.eslintrc.js

```diff
@@ -0,0 +1,59 @@
+module.exports = {
```

Comment:
There should be an entry specifying that we want to use the `mozilla` plugin: `"plugins": ["mozilla"]`